apps/website-new/docs/en/practice/monorepos/nx-for-module-federation.mdx

```diff
@@ -0,0 +1,103 @@
+- Make it Your Own: Nx is highly customizable and extensible. Fine-tune it by [creating a plugin for your organization](https://nx.dev/extending-nx/tutorials/organization-specific-plugin) or [creating a tooling plugin](https://nx.dev/extending-nx/tutorials/tooling-plugin).
+
+## Nx Module Federation Features
+
+Nx providers out of the box support for Module Federation for Angular and React projects including projects with Server Side Rendering (SSR).
+
+### Generators for Scaffolding Module Federation Projects
+
+Generators in Nx provide a consistent and convenient method for scaffolding projects in a monorepo. Nx Plugins provide generators for various frameworks and tools, including Module Federation.
+Rapidly scaffolding Module Federation projects for Angular and React requires just a single command. This is a great way to get started with Module Federation while also providing a consistent structure as you add more Producers (remotes) to your system.
+
+You can see an example of the Nx `consumer` generator below:
+```bash
+# Generate an Angular Consumer application with three Producers
+nx g @nx/angular:consumer shell --producers=products,cart,checkout
+
+# Generate a React Consumer application with three Producers
+nx g @nx/react:consumer shell --producers=products,cart,checkout
+```
+
+You can learn more about setting up Module Federation with Nx in the guides for [Angular](/practice/frameworks/angular/using-nx-for-angular) and [React](/practice/frameworks/react/using-nx-for-react).
+
+### Executors
+
+Nx provides executors for building and serving your Module Federation applications. The important one is the `module-federation-dev-server` executor which is used to serve your application locally. This executor provides the following features:
+
+- Support for serving the Consumer (host) project along with all of its Producers (remotes) in a single command with the best resource management and DX.
+- Support for serving individual Producers (remotes) that are currently being worked with HMR and file watching for faster local development.
+
+You can learn more about how we [Scale Develop Experience](./nx-for-module-federation#scaling-developer-experience-dx) for Module Federation in Nx below.
+
+By forcing you to serve the Consumer (host) project along with all of its Producers (remotes) in a single command, Nx ensures that you get immediate feedback on any changes you make to the Producer (remote) projects providing immediate feedback if it impacts other projects in the system.
+However, it also allows you to [develop as a user](./nx-for-module-federation#develop-as-a-user) such that you follow the same flow that a user would follow using the deployed application.
```

Comment:
Remove "However" so it's `It also allows...`. Sounds less negative and doesn't take anything away.